### PR TITLE
Prod make files

### PR DESCRIPTION
**Pull request by ed-sparkes**
## Link to JIRA Ticket

-

## Description

Please include a summary of the change

### Noteworthy Changes

- These are changes the reviewer should look out for

## Type of change

Delete not appropriate

- Bug fix (non-breaking change which fixes an issue)
- New feature (non-breaking change which adds functionality)
- Breaking change (fix or feature that would cause existing functionality to not work as expected)
- This change requires a documentation update

## Testing

Please tick the testing that has been completed

- [ ] Unit tests
- [ ] Integration tests

## Developer Checklist

- [ ] I have performed a self-review of my own code
- [ ] I have run the [code formatting checks](../README.md#code-quality)
- [ ] I have run the [code quality checks](../README.md#code-quality)
- [ ] New code meets [standards](https://nhsd-confluence.digital.nhs.uk/display/DI/DI+Ways+of+Working) agreed by the team
- [ ] Unit test code coverage is at or above 80%
- [ ] New and existing unit tests pass locally with my changes
- [ ] Tests have added that prove my fix is effective or that my feature works (Integration tests)
- [ ] I have made corresponding changes to the documentation
- [ ] I have cleaned down my environment (if created)

## Code Reviewer Checklist

- [ ] I have run the unit tests and they run correctly
- [ ] I can confirm the changes have been tested or approved by a tester
- [ ] I can confirm no remaining infrastructure is left over from this branch

### CODE REVIEW

File: build/automation/var/profile/demo.mk [hunk 1]
```diff
@@ -5,20 +5,21 @@
...
+DOS_API_GATEWAY_SECRETS = core-dos-uet/deployment
+DOS_API_GATEWAY_USERNAME_KEY := DOS_UET_API_GATEWAY_USER
+DOS_API_GATEWAY_PASSWORD_KEY := DOS_UET_API_GATEWAY_PASSWORD
+DOS_API_GATEWAY_URL := https://core-dos-put-ut-ddc-core-dos-api-gateway.k8s-prod.texasplatform.uk/api/change-request
+
+
 MOCK_MODE := false
 
-DB_SERVER_NAME := uec-core-dos-performance-db-12-replica-di
+DB_SERVER_NAME := uec-core-dos-put-db-12-replica-di
```
Comment (rohith1122) [inline build/automation/var/profile/demo.mk:16]: is this value should be uec-core-dos-**uet**-db-12-replica-di or uec-core-dos-put-db-12-replica-di

File: build/automation/var/profile/demo.mk [hunk 2]
```diff
@@ -5,20 +5,21 @@
...
 DB_SCHEMA := pathwaysdos
-DB_SECRET_NAME := core-dos-dev/deployment
-DB_SECRET_KEY := DB_SF_READONLY_PASSWORD
+DB_SECRET_NAME := core-dos-uet-database-upgrade/deployment
+DB_SECRET_KEY := DB_DI_READONLY_PASSWORD
 DB_USER_NAME_SECRET_NAME = uec-dos-int-dev/deployment
 DB_USER_NAME_SECRET_KEY = DOS_REPLICA_DI_USERNAME
 
-TF_VAR_ip_address_secret := uec-dos-int-dev-ip-addresses-allowlist
+TF_VAR_ip_address_secret := uec-dos-int-put-ip-addresses-allowlist
```
Comment (rohith1122) [inline build/automation/var/profile/demo.mk:25]: is this value  should be uec-dos-int-**uet**-ip-addresses-allowlist or uec-dos-int-put-ip-addresses-allowlist

**Review by rohith1122 — COMMENTED** [review]
Just double-checking the naming convention other than that its is LGTM

**Review by mabe13 — APPROVED** [review]

File: build/automation/var/profile/demo.mk (hunk 1 above)
Comment (ed-sparkes) [reply build/automation/var/profile/demo.mk:16]: is correct as is

File: build/automation/var/profile/demo.mk (hunk 2 above)
Comment (ed-sparkes) [reply build/automation/var/profile/demo.mk:25]: is correct as is

**Review by rohith1122 — APPROVED** [review]

**Pull request merged**